### PR TITLE
[chore] FAQ

### PR DESCRIPTION
**Pull request by EdGeraghty**
### Types
- [ ] Fix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Refactor (non-breaking change which improves code quality - QA thoroughly )
- [ ] Breaking Change (fix or feature that would cause existing functionality to not work as expected)
- [x] Chore (updates that would not affect or change the code involving the app itself)

#### 🔮 Features
FAQ
### QA

- [ ] Signup
- [ ] Login
- [ ] Logout
- [ ] Unencrypted Chat 
- [ ] Encrypted Chat 
- [ ] Group Chats
- [ ] Profile Changes
- [ ] Theming Changes
- [ ] Force Kill and Restart

### Final Checklist
 
- [ ] All associated issues have been linked to PR
- [ ] All necessary changes made to the documentation
- [ ] Parties interested in these changes have been notified

### CODE REVIEW

**Comment by ereio** [comment]
Awesome work! Would be great to re-use this same copy in a FAQ page on syphon.org as well

**Review by ereio — APPROVED** [review]

File: docs/faq-en.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Syphon is currently in early development. Whilst we believe the code to be of a standard where we can share with a wider userbase, Syphon does not rely on public Matrix libraries, instead coding against [the Matrix specification](https://spec.matrix.org/latest/).
+  
+This means that Syphon is currently missing features found in other Matrix chat clients as the development team work to implement them.
+
+[Syphon's feature roadmap can be found here](https://syphon.org/roadmap).
+
+## [X] feature is acting strange/my settings aren't saved
+
+Syphon is currently in open alpha, meaning development is currently rapid meaning each release consists of quite a "jump" in features.
```
Comment (0x1a8510f2) [inline docs/faq-en.md:13]: >meaning development is currently rapid meaning

Sounds a little odd IMO. Perhaps "meaning development is currently rapid so"?

**Review by 0x1a8510f2 — DISMISSED** [review]
One small tweak but LGTM otherwise!

**Review by 0x1a8510f2 — DISMISSED** [review]
LGTM!

**Review by dnisbetjones — APPROVED** [review]

**Review by ereio — APPROVED** [review]

**Pull request merged**